http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```diff
@@ -139,8 +141,7 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
     runtimeAndRuntimeConfig: RuntimeAndRuntimeConfig,
     dataprocInstances: Set[DataprocInstance], //only applies to dataproc
     message: Option[String],
-    ip: Option[IP] = None,
-    toolsToCheck: List[RuntimeImageType] = List.empty // If empty, then check all tools; if defined, then only check specified tools
+    ip: Option[IP] = None
```

Comment:
previous `toolsToCheck` is an optimization that allows us not to re-check a tool if we've already checked it, but now that we're using `streamFUntilDone`, we don't need this anymore